### PR TITLE
chore(release): release 4.4.0-rc1

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Changelog for release 4.4.0-rc1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,209 @@
...
+  * We also changed the integration mechanism with
+    [ScanCode](https://scancode-toolkit.readthedocs.io) resulting in major
+    speed improvements in the scan.
+* With this release, we also bring support for Debian Bookworm (12)
+* Support extraction of [Zstandard](https://www.zstd.net) files
+* Support GitHub Actions in the scanner image and generate SPDX reports
+* Multiple fixes in SPDX reports
+* Sync with SPDX License list v3.22
+
+#### Credits to contributors for 4.3.0-rc1
```
Comment (shaheemazmalmmd) [inline CHANGELOG.md:33]:
```suggestion
#### Credits to contributors for 4.4.0-rc1
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,209 @@
...
+
+From the GIT commit history, we have the following contributors since
+[4.2.1](https://github.com/fossology/fossology/releases/tag/4.2.1):
```
Comment (shaheemazmalmmd) [inline CHANGELOG.md:36]:
```suggestion
[4.3.0](https://github.com/fossology/fossology/releases/tag/4.3.0):
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,209 @@
 -->
 # Changelog of FOSSology
 
+### 4.4.0-rc1 (Nov 21st 2023)
+
+This release adds important corrections to
+[4.3.0](https://github.com/fossology/fossology/releases/tag/4.3.0)
+
+The release 4.4.0-rc1 introduces a number of corrections to
```
Comment (shaheemazmalmmd) [inline CHANGELOG.md:12]:
```suggestion
The release [4.4.0-rc1](https://github.com/fossology/fossology/releases/tag/4.4.0-rc1) introduces a number of corrections to
```

**Review by shaheemazmalmmd — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (GMishx) [reply CHANGELOG.md:12]: We never provided this link in the changelogs before. Do you want me to include them?

**Pull request merged**